.cicd/resource_interface_mappings.py

```diff
@@ -0,0 +1,108 @@
+# #######
+# Copyright (c) 2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from openstack import connect
+
+
+class InterfaceBase(object):
+
```

Comment:
@EarthmanT Do you think adding these class variables 
```
    type_name = None
    client_name = None
    get_method_name =None
    delete_method_name = None
```
Would be good to give an indication that any class want to extend this interface must override these variables ?